cishouseholds/derive.py

```diff
@@ -1168,17 +1168,14 @@ def assign_school_year(
         )
         .withColumn(
             column_name_to_assign,
-            F.floor(F.datediff(F.col(reference_date_column), F.col("school_start_date")) / 365.25).cast("integer"),
+            F.floor(F.round(F.datediff(F.col(reference_date_column), F.col("school_start_date")) / 365.2425, 3)).cast(
```

Comment:
This is changed to increase accuracy I assume, wasn't the function exact_year_diff defined below to be used? If I'm misunderstanding then np